Extension/package.json

```diff
@@ -42,6 +42,168 @@
+            "when": "isWindows",
+            "media": {
+              "altText": "markdown file with instructions for configuring default build task on MSVC with Windows",
+              "path": "walkthrough/build-project-windows.md"
+            }
+          },
+          {
+            "id": "run.project.mac",
+            "title": "Run and debug your C++ file",
+            "description": "Create a debug configuration for your project. \n[Learn more](https://code.visualstudio.com/docs/cpp/config-clang-mac#_debug-helloworldcpp)",
+            "when": "isMac",
+            "media": {
+              "altText": "markdown file with instructions for starting a debugging session on macOS",
+              "path": "walkthrough/run-and-debug-project-mac.md"
+            }
+          },
+          {
+            "id": "run.project.linux",
+            "title": "Run and debug your C++ file",
+            "description": "Create a debug configuration for your project. \n[Learn more](https://code.visualstudio.com/docs/cpp/config-linux#_debug-helloworldcpp)",
+            "when": "isLinux",
+            "media": {
+              "altText": "markdown file with instructions for starting a debugging session on Linux",
+              "path": "walkthrough/run-and-debug-project-linux.md"
+            }
+          },
+          {
+            "id": "run.project.windows",
+            "title": "Run and debug your C++ file",
+            "description": "Create a debug configuration for your project. \n[Learn more](https://code.visualstudio.com/docs/cpp/config-msvc#_debug-helloworldcpp)",
+            "when": "isWindows",
+            "media": {
+              "altText": "markdown file with instructions for starting a debugging session on Windows",
+              "path": "walkthrough/run-and-debug-project-windows.md"
+            }
+          },
+          {
+            "id": "install.cmake.tools",
+            "title": "Install CMake Tools",
+            "description": "Do you build your project with CMake? Install the CMake Tools extension to seamlessly build and debug CMake project. \n[Install CMake Tools](command:workbench.extensions.installExtension?%22ms-vscode.cmake-tools%22)",
+            "doneOn": {
+              "command": "workbench.extensions.installExtension"
+            },
+            "media": {
+              "altText": "screenshot of CMake Tools extension page",
+              "path": {
+                "light": "walkthrough/cmake-tools-light.png",
+                "dark": "walkthrough/cmake-tools-dark.png",
+                "hc": "walkthrough/cmake-tools-hc.png"
+              }
+            }
+          },
+          {
+            "id": "watch.tutorials",
+            "title": "Lean back and get started",
+            "description": "Watch this series of short and practical videos about setting up C++ IntelliSense, building, and debugging C++ projects. \n[Watch Tutorials](https://code.visualstudio.com/docs/cpp/introvideos-cpp)",
```

Comment:
Should be something like "setting up C++ IntelliSense and building and debugging C++ projects", "setting up C++ IntelliSense, building and debugging C++ projects, and <some other topic>", or "configuring, building, and debugging C++ projects". The reason is because in original text the comma is used as delimiter of sequence, but the 2nd item, "building" is detached from the required object, "C++ projects".

Reply:
Yeah, I struggled with that comma placement because I was worried that "and building and debugging" looked weird. But I agree that "setting up C++ IntelliSense and building and debugging C++ projects" is better. I'll change this